File: Codewars/Javascript/8_kyu/Reversed sequence.js
// DESCRIPTION:
// Build a function that returns an array of integers from n to 1 where n>0.

// Example : n=5 --> [5,4,3,2,1]

// My Solution:
const reverseSeq = n => {
    let res = []
    while (n >= 1) {
        res.push(n)
        n -= 1
    }
    return res;
};

File: Codewars/Go/8_kyu/Reversed sequence.go
// Description:
// Build a function that returns an array of integers from n to 1 where n>0.

// Example : n=5 --> [5,4,3,2,1]

// My Solution:
package kata

func ReverseSeq(n int) (res[]int) {
  for idx:= n; idx > 0; idx--{
    res = append(res, idx)
  }
  return res
}